backend/pkg/modules/internal/differ/http_handler.go

```diff
@@ -0,0 +1,24 @@
+package differ
+
+import (
+	"net/http"
+
+	oapicommon "github.com/openclarity/apiclarity/api3/common"
+	"github.com/openclarity/apiclarity/backend/pkg/modules/internal/common"
+)
+
+type httpHandler struct{}
+
+func (h *httpHandler) GetVersion(w http.ResponseWriter, r *http.Request) {
+	common.HttpResponse(w, http.StatusOK, &oapicommon.ModuleVersion{
+		Version: "1",
```

Comment:
what is this version for?

Reply:
not sure I think all modules should implement this I will check

Reply:
removed